hunting/generate_markdown.py

```diff
@@ -0,0 +1,87 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+
+"""Lightweight builtin toml-markdown converter."""
+
+import tomllib
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List, Optional
+
+HUNTING_DIR = Path(__file__).parent
+
+
+@dataclass
+class Hunt:
+    """Dataclass to represent a hunt."""
+
+    author: str
+    datasource: str
+    uuid: str
+    name: str
+    language: str
+    query: str
+    notes: Optional[List[str]] = field(default_factory=list)
+    mitre: Optional[List[str]] = field(default_factory=list)
+    references: Optional[List[str]] = field(default_factory=list)
+
+
+@dataclass
+class TomlHunt:
+    """Dataclass to represent a TOML hunt."""
+
+    hunt: Hunt
+
+
+def load_toml(contents: str) -> Hunt:
+    """Load and validate TOML content as Hunt dataclass."""
+    toml_dict = tomllib.loads(contents)
+    return TomlHunt(hunt=Hunt(**toml_dict["hunt"])).hunt
```

Comment:
if you are just going to immediately return `TomlHunt.hunt` I would stip out `TomlHunt` altogether from this